websites/W/wumpus.store/presence.ts

```diff
@@ -0,0 +1,57 @@
+const presence = new Presence({
+		clientId: "998532462755979275",
+	}),
+	browsingUnix = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://imgur.com/QCtoeEM.png",
+		},
+		{ pathname } = document.location;
+	presenceData.details = "Viewing Home Page";
+
+	let title: HTMLElement, logo: HTMLElement;
+	presenceData.startTimestamp = browsingUnix;
+
+	switch (pathname) {
+		case "/bot/add": {
+			presenceData.details = "Adding a new bot";
+			break;
+		}
+		case "/search": {
+			presenceData.details = "Searching for a new bot";
+			break;
+		}
+		case "/": {
+			presenceData.details = "Viewing Home Page";
+			break;
+		}
+	}
+
+	if (
+		pathname.startsWith("/bot/") &&
+		document.querySelector(".css-1igwmid > b")
+	) {
+		title = document.querySelector(".css-1igwmid > b");
```

Comment:
This selector may be potentially unstable. Is there an alternate selector that doesn't use compiled css names?